docs/en/operations/settings/settings-formats.md

```diff
@@ -634,6 +634,17 @@ Ignore extra columns in rows with more columns than expected and treat missing c
 
 Disabled by default.
 
+### output_format_markdown_escape_special_characters {#output_format_markdown_escape_special_characters}
+
+When enabled, automatically escape special characters in Markdown.
```

Comment:
"automatically" is redundant.

Let's mention which character is used for escaping (\\)

Please also provide a link / reference which characters are considered "special" (see my other comment about CommonMark).